packages/esm-patient-registration-app/src/patient-registration/field/gender/gender-field.component.tsx

```diff
@@ -20,10 +24,14 @@ export const GenderField: React.FC = () => {
       <div className={styles.sexField}>
         <p className="cds--label">{t('genderLabelText', 'Sex')}</p>
         <RadioButtonGroup name="gender" orientation="vertical" onChange={setGender} valueSelected={field.value}>
-          <RadioButton id="male" labelText={t('maleLabelText', 'Male')} value="Male" />
-          <RadioButton id="female" labelText={t('femaleLabelText', 'Female')} value="Female" />
-          <RadioButton id="other" labelText={t('otherLabelText', 'Other')} value="Other" />
-          <RadioButton id="unknown" labelText={t('unknownLabelText', 'Unknown')} value="Unknown" />
+          {fieldConfigs.map((option) => (
+            <RadioButton
+              key={option.label}
+              id={option.id}
```

Comment:
@makombe This is a pointless configuration field; please do not make implementers fill in configuration fields needlessly. You can use anything for `id`, it doesn't matter, as long as it's different for each instance.